docs/resources/device.md

```diff
@@ -73,6 +73,10 @@ resource "netbox_device" "test" {
 - `status` (String) Valid values are `offline`, `active`, `planned`, `staged`, `failed` and `inventory`. Defaults to `active`.
 - `tags` (Set of String)
 - `tenant_id` (Number)
+- `vc_master` (Boolean) Required when `vc_master` and `virtual_chassis_id` is set.
+- `vc_position` (Number)
+- `vc_priority` (Number)
+- `virtual_chassis_id` (Number) Required when `vc_master` and `virtual_chassis_id` is set.
 
```

Comment:
I do not like the naming of the `vc_X` attributes. Can we rename them to `virtual_chassis_X`?
I know this is directly taken from the API, but the API naming is kinda stupid in my opinion :D

Reply:
Sure, will change that.

Reply:
I've adjusted the attributes.